src/index.tsx

```diff
@@ -101,6 +103,12 @@ function Wordcloud({
         .sort((x, y) => descending(x.value, y.value))
         .slice(0, maxWords);
 
+
+      const pmr = require('pseudo-math-random');
+      const randomGenerator: ()=> number = options.deterministic ? pmr('deterministic') : pmr();
```

Comment:
You can fix a few code formatting issues by running `yarn lint`.  It should fix up most of the formatting with the prettier rules in the codebase.

Reply:
You can probably call this `random`, since it effectively is a callback replacing `Math.random` in the code here.